js/main.js

```diff
@@ -6,5 +6,139 @@ const displayMobileMenu = () => {
+  technologies: ['html', 'Ruby on rails', 'css', 'javascript'],
+  popupTech: ['html', 'css', 'javaScript', 'github', 'Ruby', 'Bootstrap'],
+  image: './images/facebook-360.svg',
+  imageMobile: './images/tonic-portfolio-two.svg',
+  imagePopup: './images/facebook-360.svg',
+  imageMobilePopup: './images/tonic-portfolio-two.svg',
+  seeLive: 'https://saidrasinlic.github.io/MicroversePortfolio/',
+  seeSource: 'https://github.com/SaidRasinlic/MicroversePortfolio',
+},
+{
+  title: 'Uber Navigation',
+  exprience: ['UBER', 'Lead Developer', '2018'],
+  desription: 'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+  popDescription: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea.',
+  technologies: ['html', 'Ruby on rails', 'css', 'javascript'],
+  popupTech: ['html', 'css', 'javaScript', 'github', 'Ruby', 'Bootstrap'],
+  image: './images/uber-nav.svg',
+  imageMobile: './images/multi-portfolio-two.svg',
+  imagePopup: './images/uber-nav.svg',
+  imageMobilePopup: './images/multi-portfolio-two.svg',
+  seeLive: 'https://saidrasinlic.github.io/MicroversePortfolio/',
+  seeSource: 'https://github.com/SaidRasinlic/MicroversePortfolio',
+},
+];
```

Comment:
- [OPTIONAL]
All the popups shows the same content `Tonic`. While I see in your `projectDetail` table other contents (ex: 'Facebook 360',  'Uber Navigation'). Kindly configure your pop-up content so it displays the content of the clicked project.